File: 00/0048/0048.py
x = int(input())
y = int(input())
l = int(input())
ans = (abs(x) + l - 1) // l + (abs(y) + l - 1) // l
if y >= 0 and x != 0:
	ans += 1
elif y < 0:
	ans += 2
print(ans)

File: 00/0048/0048.cpp
#include <iostream>
using namespace std;

int main() {
	int x, y, l;
	cin >> x >> y >> l;
	int ans = (abs(x) + l - 1) / l + (abs(y) + l - 1) / l;
	if (y >= 0 && x != 0) ans++;
	else if (y < 0) ans += 2;
	cout << ans << '\n';
}